src/frontend/src/btc/utils/btc-transactions.utils.ts

```diff
@@ -35,11 +75,11 @@ export const mapBtcTransaction = ({
 	return {
 		id: hash,
 		timestamp: BigInt(time),
-		value: nonNullish(output?.value) ? BigInt(output.value) : undefined,
+		value: nonNullish(value) ? BigInt(isTypeSend ? value + utxosFee : value) : undefined,
 		status,
 		blockNumber: block_index ?? undefined,
 		type: isTypeSend ? 'send' : 'receive',
 		from: isTypeSend ? btcAddress : inputs[0].prev_out.addr,
-		to: nonNullish(output) ? (isTypeSend ? output.addr : btcAddress) : undefined
+		to
```

Comment:
same question here for `type`, `from` and `to`: i imagine that the transactions that arrive in this function are either send from me or receive to me, and not unrelated

Reply:
correct, it will be a transaction related to your current BTC address